Misc/NEWS.d/next/Tests/2024-10-16-14-33-57.gh-issue-125584.5goH3R.rst

```diff
@@ -0,0 +1,2 @@
+Added the missing requirement of ``network`` resource for
+``test.test_urllib2.HandlerTests.test_ftp_error``.
```

Comment:
We can omit the `NEWS` entry from this PR. This only affects tests, and there is only one release with that bug, which is `3.14a1`.

Reply:
Sure, removed.